postprocessing.ipynb

```diff
@@ -0,0 +1,1296 @@
+       "       3.5975e+02], dtype=float32)</pre></div></li><li class='xr-var-item'><div class='xr-var-name'><span class='xr-has-index'>latitude</span></div><div class='xr-var-dims'>(latitude)</div><div class='xr-var-dtype'>float32</div><div class='xr-var-preview xr-preview'>90.0 89.75 89.5 ... -89.75 -90.0</div><input id='attrs-ddba245f-ede2-4c43-afef-a8ce9a7abb1a' class='xr-var-attrs-in' type='checkbox' ><label for='attrs-ddba245f-ede2-4c43-afef-a8ce9a7abb1a' title='Show/Hide attributes'><svg class='icon xr-icon-file-text2'><use xlink:href='#icon-file-text2'></use></svg></label><input id='data-6b95d463-59a6-4cd9-86f7-e54c958accb2' class='xr-var-data-in' type='checkbox'><label for='data-6b95d463-59a6-4cd9-86f7-e54c958accb2' title='Show/Hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-var-attrs'><dl class='xr-attrs'><dt><span>units :</span></dt><dd>degrees_north</dd><dt><span>long_name :</span></dt><dd>latitude</dd></dl></div><div class='xr-var-data'><pre>array([ 90.  ,  89.75,  89.5 , ..., -89.5 , -89.75, -90.  ], dtype=float32)</pre></div></li><li class='xr-var-item'><div class='xr-var-name'><span class='xr-has-index'>time</span></div><div class='xr-var-dims'>(time)</div><div class='xr-var-dtype'>datetime64[ns]</div><div class='xr-var-preview xr-preview'>2021-03-23 ... 2021-03-25T01:00:00</div><input id='attrs-2e90eb7d-55d1-4a0d-a34c-36a3c0559f5b' class='xr-var-attrs-in' type='checkbox' ><label for='attrs-2e90eb7d-55d1-4a0d-a34c-36a3c0559f5b' title='Show/Hide attributes'><svg class='icon xr-icon-file-text2'><use xlink:href='#icon-file-text2'></use></svg></label><input id='data-8a356a2e-d197-45aa-b1bb-c44503fba67f' class='xr-var-data-in' type='checkbox'><label for='data-8a356a2e-d197-45aa-b1bb-c44503fba67f' title='Show/Hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-var-attrs'><dl class='xr-attrs'><dt><span>long_name :</span></dt><dd>time</dd></dl></div><div class='xr-var-data'><pre>array([&#x27;2021-03-23T00:00:00.000000000&#x27;, &#x27;2021-03-23T01:00:00.000000000&#x27;,\n",
+       "       &#x27;2021-03-23T02:00:00.000000000&#x27;, &#x27;2021-03-23T03:00:00.000000000&#x27;,\n",
```

Comment:
May I ask if this code is generated automatically? I see the date here is very specific. How do we update the date if we are using a different period of time>

Reply:
This is a Jupyter notebook file, and you need to open it using Jupyter. It can easily be updated to an arbitrarily chosen date. We can talk about it tomorrow.